conf/generators/templates/encapsulation-fail.json

```diff
@@ -36,9 +36,6 @@
 		["local_from_outside", "1", "b", "b_loc", "simulation"],
 		["inherited_prot_from_outside", "1", "b", "a_prot", "simulation"],
 		["prot_from_outside", "1", "b", "b_prot", "simulation"],
-		["inherited_local_from_inside", "1", "a_loc", "b", "simulation"],
-		["local_from_inside", "0", "b_loc", "b", "simulation parsing"],
-		["inherited_prot_from_inside", "0", "a_prot", "b", "simulation parsing"],
-		["prot_from_inside", "0", "b_prot", "b", "simulation parsing"]
+		["inherited_local_from_inside", "1", "a_loc", "b", "simulation"]
 	]
```

Comment:
We can now remove the 2nd and last columns and adjust the template above to always set `should_fail: 1` and `type: simulation`

Reply:
I think it would be better if we replaced the second column with the reason for 
`should_fail_because` and add the tag `should_fail_because` to the template array

Reply:
I think that for now we can just left this as `should_fail: 1` and create a separate PR for `should_fail_because` because this PR is already pretty big